features/26-mfa/api.md

```diff
@@ -500,24 +526,42 @@ interface ActivateOOBResult {
 }
 
 async function createNewOOB(options: CreateNewOOBOptions): Promise<CreateNewOOBResult>;
-async function triggerOOB(authenticatorID?: string): Promise<void>;
-async function activateOOB(authenticatorID: string, code: string): Promise<ActivateOOBResult>;
+async function triggerOOB(options?: TriggerOOBOptions): Promise<void>;
+async function activateOOB(opitons: ActivateOOBOptions): Promise<ActivateOOBResult>;
 
 // Error inspection
 
-function isMFARequiredError(err: unknown): boolean;
+interface AuthenticationSession {
+  token: string;
+  step: "identity" | "mfa";
+}
+
+function decodeAuthenticationSession(value: unknown): AuthenticationSession | undefined;
 
 // Authenticate
 
-interface MFAAuthenticateOptions {
+interface AuthenticateWithTOTPOptions {
+  otp: string;
+  authenticationSessionToken: string;
```

Comment:
I think the authentication session token can be handled automatically by SDK? So something like `currentAuthenticationStep` would be exposed, and developer does not need to pass the token around the different screens.

Reply:
The container is already very stateful. One of my experience on using the v1 SDK is that the record API is full of mutation. Maybe we do not expose the token but require the developer to pass in the authentication session. The authentication session in the SDK is like a opaque handle.